src/NzbDrone.Core/Indexers/IndexerFactory.cs

```diff
@@ -62,12 +67,31 @@ public List<IIndexer> SearchEnabled(bool filterBlockedIndexers = true)
 
             if (filterBlockedIndexers)
             {
-                return FilterBlockedIndexers(enabledIndexers).ToList();
+                return FilterBlockedIndexers(SubstituteIndexerUrls(enabledIndexers)).ToList();
             }
 
             return enabledIndexers.ToList();
         }
 
+        private IEnumerable<IIndexer> SubstituteIndexerUrls(IEnumerable<IIndexer> indexers)
```

Comment:
I didn't want Factory to change settings, only overrule them but UrlBlacklist can only be done here. The substitute would actually be persisted by a periodic task, Housekeeping was my thought.